.github/workflows/release.yml

```diff
@@ -73,8 +73,6 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-        with:
```

Comment:
shouldn't we have the tag already created?


```
      - name: Create GitHub release
        uses: actions/create-release@v1
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
        with:
          tag_name: v${{env.version_number}}
```

Reply:
yes, but version_number is retrieved in one of the next step

checkout with specific tag is not needed here, since all packages are built from scratch in `Release to pypi` step